airflow/utils/email.py

```diff
@@ -68,6 +73,8 @@ def send_email(
 
 
 def send_email_smtp(
+    from_email: str,
+    from_name: str,
```

Comment:
This isn't backwards compatible as the positional args have been changed.

So if someone had it as below:

```
def send_email_smtp("to@example.org", "[Some Subject]", html_content="xyz")
```

This will now not work

Reply:
We should add it at the bottom after `conn_id`